src/expr.h

```diff
@@ -518,7 +518,6 @@ class ConstExpr : public Expr {
         varying. */
     int Count() const;
 
-  private:
     AtomicType::BasicType getBasicType() const;
 
     const Type *type;
```

Comment:
I'm also not sure why the `union` below needs to be not private, I think it's not used outside of the class with these changes.

Reply:
Oops, it is an artifact of the initial version that accessed its members directly.